bootstrap.ps1

```diff
@@ -0,0 +1,10 @@
+if ( !(Join-Path 'src' 'nimph.nim' | Test-Path) ) {
```

Comment:
I've ported the original script as is, but this line is weird. Why are we checking against src/nimph.nim when we really need to check whether the repos has been cloned yet? Shouldn't we check against nimph/src/nimph.nim?